docs/src/content/docs/challenges/rxjs/49-hold-to-send-btn.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 🟠 Hold to save button
+description: You're tasked with implementing Lucie's button design, requiring holding it for a set time to save, taking over from Sacha; functionalities include configuring duration, countdown initiation on "mousedown", progress bar reset on "mouseleave" or "mouseup", reflecting remaining time, and simulating save request on hold completion, using RxJS operators and ensuring declarative code.
+author: timothy-alcaide
+contributors:
+  - alcaidio
+challengeNumber: 49
+command: rxjs-hold-to-save-btn
+sidebar:
+  order: 19
+  badge: New
+---
+
+## Context
+
+As a member of the development team, you have to respond to a specific request from the UX designer, 👩🏻‍🎨 Lucie, who has designed a button that must be held down for X amount of time to save a save request.
+
+Sacha 👶🏼 the trainee has already integrated the design but doesn't know how to perform the "holdable" functionality.
+
+So you're going to take over from him.
+
+## Functional expectation
+
+> "As a user, I would like to save something by holding down the button for a certain amount of time."
+
+Here the prototype made by Lucie:
+
+![prototype gif](../../../../../../docs/src/assets/rxjs/49/prototype.gif)
```

Comment:
The build fail, because the path is wrong. 

try npm run doc:dev before pushing to test that the documentation site is launching correctly